harness/determined/deploy/gcp/cli.py

```diff
@@ -212,6 +216,13 @@ def handle_up(args: argparse.Namespace) -> None:
                             "'ip-address:/file-share'; if not provided, a new Filestore "
                             "instance will be created",
                         ),
+                        Arg(
```

Comment:
It would probably be a bit cleaner here to do something like:
`Arg(
     "--no-filestore",
      help="whether to create a new Filestore if no filestore address is provided",
      action="store_true",
),`

Functionally works the same, but makes it clearer that it is an optional parameter in my view.